### PR TITLE
Adding bahasa indonesia

### PR DESCRIPTION
**Pull request by mirzaakhena**
I have added bahasa indonesia language to jhipster, please review. Thanks

### CODE REVIEW

**Comment by deepu105** [comment]
You need to add the languages into https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angular/src/main/webapp/app/shared/language/_find-language-from-key.pipe.ts and https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angularjs/src/main/webapp/app/components/language/_language.filter.js

**Comment by jdubois** [comment]
Thanks! I added it to our documentation page

**Comment by mirzaakhena** [comment]
You are welcome @jdubois , by the way when will i can see the change?

**Comment by jdubois** [comment]
The documentation should already by live (it's automatic), for the generator it will be in the next release

**Comment by deepu105** [comment]
@jdubois but shouldnt we update docs only after release?

File: generators/generator-base.js [hunk 1]
```diff
@@ -472,6 +472,7 @@ module.exports = class extends PrivateBase {
             { name: 'Greek', value: 'el' },
             { name: 'Hindi', value: 'hi' },
             { name: 'Hungarian', value: 'hu' },
+            { name: 'Bahasa Indonesia', value: 'id' },
```
Comment (deepu105) [inline generators/generator-base.js]: here it should be called Indonesian. I'll change it

**Comment by jdubois** [comment]
@deepu105 yes, but I was lazy...

File: generators/generator-base.js (hunk 1 above)
Comment (sendilkumarn) [inline generators/generator-base.js]: It is either bahasa indonesia or indonesian right ?

Comment (mirzaakhena) [inline generators/generator-base.js]: bahasa indonesia

Comment (mirzaakhena) [inline generators/generator-base.js]: let me fixed it..

Comment (deepu105) [inline generators/generator-base.js]: Yes but in our intenal file we use inky english name for language and in english it is Indonesian. I checked Wikipedia

Comment (deepu105) [inline generators/generator-base.js]: Anyway I have changed it here already. In the UI however it will be bahasa Indonesia

Comment (mirzaakhena) [inline generators/generator-base.js]: Sometime people say 'Bahasa', sometime 'Indonesian' and sometime 'Bahasa Indonesia'. 
'Bahasa' means 'Language' in english.

**Comment by cbot59** [comment]
How do I contribute to this? As I think it needs some consistency on translation and fix a little typo.

**Comment by ruddell** [comment]
@cbot59 Feel free to open a Pull Request with the changes.  You can look at the [Submitting a Pull Request](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md#submitting-a-pull-request) section of the Contributing guide if you need more info

**Pull request merged**